taccsite_cms/settings.py

```diff
@@ -490,6 +490,7 @@ def gettext(s): return s
     # core TACC CMS
     # HELP: If this were top of list, would TACC/Core-CMS/pull/169 fix break?
     'taccsite_cms',
+    'apps.email_management',
```

Comment:
**Reminder to Self.** This is the first standard app since the creation of custom apps feature. I think it does belongs in `apps.`, just like this. And it highlights the old `taccsite_cms` apps — good; I want that to raise questions; they need to be retired.

**Thinking Aloud.** Should this app be in its own repo, like [`djangocms_tacc_image_gallery`](https://github.com/TACC/Core-CMS-Plugin-Image-Gallery)? If so, no rush. It's fine as is, and moving it would be backwards compatibile.

Reply:
I think, maybe this can go in another exploratory ticket.

Reply:
Yes, indeed. I'm just typing it out in context, so anyone later reviewing how this new app was done can see it is intended manner, not a one-off.